Tests/FileIO/TestXmlGmlReader.cpp

```diff
@@ -209,7 +209,7 @@ TEST_F(FileIOXmlGml, QtXmlGmlWriterReaderTest)
 {
 	// Writer test
 	std::string test_data_file(BaseLib::BuildInfo::tests_tmp_path
-		+ boost::filesystem::unique_path().string() + ".gml");
```

Comment:
I understand the reason to change the code here, but I like the `unique_path` feature also. I opened an issue #942 for further discussion separate from this PR.